com.github.xournalpp.xournalpp.yaml

```diff
@@ -90,11 +95,9 @@ modules:
 
```

Comment:
Build works with CMake: Take a look at [this script](https://github.com/xournalpp/xournalpp/blob/mac-ci-builds/mac-setup/build-sndfile.sh). Even though it builds for MacOS it does work flawlessly.